package/carbonio-mta

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+SERVICE_NAME="carbonio-mta"
+
+if [[ $(id -u) -ne 0 ]]; then
+  echo "Please run as root"
+  exit 1
+fi
+
+if [[ "$1" != "setup" ]]; then
+  echo "Syntax: ${SERVICE_NAME} <setup> to automatically setup the service"
+  exit 1;
+fi
+
+# decrypt the bootstrap token, asking the password to the sys admin
+# --setup check for SETUP_CONSUL_TOKEN env. variable and uses it
+# to avoid re-asking for the password multiple times
+echo -n "insert the cluster credential password: "
+export CONSUL_HTTP_TOKEN=$(service-discover bootstrap-token --setup)
+EXIT_CODE="$?"
+echo ""
+if [[ "${EXIT_CODE}" != "0" ]]; then
+  echo "cannot access to bootstrap token"
```

Comment:
> "cannot access to bootstrap token"

Did you mean "cannot access the bootstrap token"?

Reply:
Yeah, the sentence is not syntactically correct as you pointed out.
The script instructions are the same as other (see mailbox), but will for sure update it if it improves readability!
In my opinion we should find a way to make these steps DRY across services where possible.

Thanks :)

Reply:
@keshavbhatt , now that I read it better and in the context, I think the meaning is that it is "not possible to access to the bootstrap-token service/command". That is why it's "to" and not "the", because the latter would mean access to "the token".

Reply:
Then it would make more sense to write "Unable to access bootstrap token" However, I will leave you with the decision. I personally will reconsider changing this elsewhere as well :)

Reply:
also consider "Unable to get bootstrap token" since this is exactly what we are doing in that subshell.

Reply:
Thanks @keshavbhatt , I am making some changes because there are already new guidelines to follow (see https://github.com/Zextras/consul-templates/) and I will stick to those descriptions